package/src/components/MessageInput/__tests__/UploadProgressIndicator.test.js

```diff
@@ -32,14 +32,14 @@ describe('UploadProgressIndicator', () => {
     });
   });
 
-  it('should render an inactive RETRY UploadProgressIndicator', async () => {
+  it('should render an inactive UploadProgressIndicator', async () => {
     const action = jest.fn();
 
     const { queryByTestId, toJSON } = render(
       <ThemeProvider>
         <UploadProgressIndicator
           action={action}
-          type={ProgressIndicatorTypes.RETRY}
+          type={ProgressIndicatorTypes.INACTIVE}
         ></UploadProgressIndicator>
       </ThemeProvider>,
     );
```

Comment:
Do we know the original intent here? Or are we just changing it to make the test green? I don't feel safe merging this change without understanding why it's made

Reply:
Looking at the test we are testing for the inactive component and not retry (given the id: `testID='inactive-upload-progress-indicator'`). We are testing for an inactive state here and not retry which has a different testID @madsroskar

Reply:
I'll talk to Vish about this to get more context and also compare the code before and after the refactor

Reply:
✅